### PR TITLE
Test results widget

### PR DESCRIPTION
**Pull request by Prevole**

### CODE REVIEW

**Comment by Prevole** [comment]
Not ready yet but a pre-review will be appreciated. The commits is an attempt to consolidate the labels for tickets, tags, categories, project name and version in a common way across the whole application.
New directives to render those data are available with coherent styling for all widgets and pages.

**Comment by Prevole** [comment]
Travis CI is failing because boot.js.erb is not yet commited. Work in progress.

**Comment by Prevole** [comment]
Improvements to make sure responsiveness is working. Also working with Safari, Firefox and Chrome.

File: client/boot.js.erb [hunk 1]
```diff
@@ -70,11 +70,11 @@ angular.module('probedock', [
   'probedock.testSuiteSizeWidget',
   'probedock.userDetailsWidget',
   // components
+  'probedock.dataLabel',
```
Comment (AlphaHydrae) [inline client/boot.js.erb]: All other components declare one directive (I think). So this one should be named `dataLabels` to indicate that it declares multiple directives.

File: client/components/data-label/label.directive.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
+function linkFn(type) {
```
Comment (AlphaHydrae) [inline client/components/data-label/label.directive.js]: I think these functions can be put at the bottom of the file without change in functionality. I would prefer to see the `angular.module` call at the top if it works.

File: client/components/data-label/project-version-label.template.html.slim [hunk 3]
```diff
@@ -0,0 +1,7 @@
+span.test-data-label
+  '
+  span.label.label-primary ng-if='organization && project'
+    a.label-link ng-if='linkable' href='#' ui-sref='org.projects.show({ orgName: organization.name, projectName: project.name })' {{ project | projectName }} {{ projectVersion }}
+    span ng-if='!linkable' {{ project | projectName }} {{ projectVersion }}
+  span.label.label-primary ng-if='!organization || !project'
```
Comment (AlphaHydrae) [inline client/components/data-label/project-version-label.template.html.slim]: Do we want to display anything if this condition is true?

File: client/components/data-label/label.directive.js [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+    },
+    template: '<div class="test-key-label" ng-class="{copied: copied}" clip-copy="key.key || key" clip-click="onCopied({ key: key })" tooltip="Click to copy">{{ key.key || key }}</div>'
+  };
+}).directive('apiIdLabel', function() {
+  return {
+    restrict: 'E',
+    scope: {
+      apiId: '='
+    },
+    template: '<div class="api-id-label" clip-copy="apiId">{{ apiId }}</div>'
```
Comment (AlphaHydrae) [inline client/components/data-label/label.directive.js]: There should be a `Click to copy` tooltip.

File: client/widgets/recent-activity/activity.css.styl [hunk 5]
```diff
@@ -7,9 +7,10 @@
   .singleRunner
     .name
       margin-left 0.2em
-  .singleRunner, .multipleRunners
-    span
-      vertical-align middle
+  span.singleRunner, span.multipleRunners
```
Comment (AlphaHydrae) [inline client/widgets/recent-activity/activity.css.styl]: While you're at it, please rename these classes with the latest CSS naming convention, i.e. `single-runner` and `multiple-runners`.

File: client/boot.js.erb (hunk 1 above)
Comment (Prevole) [inline client/boot.js.erb]: Done.

File: client/components/data-label/label.directive.js (hunk 2 above)
Comment (Prevole) [inline client/components/data-label/label.directive.js]: Done.

File: client/components/data-label/project-version-label.template.html.slim (hunk 3 above)
Comment (Prevole) [inline client/components/data-label/project-version-label.template.html.slim]: Used to show only the version without the project name.

File: client/components/data-label/label.directive.js (hunk 4 above)
Comment (Prevole) [inline client/components/data-label/label.directive.js]: Done.

File: client/widgets/recent-activity/activity.css.styl (hunk 5 above)
Comment (Prevole) [inline client/widgets/recent-activity/activity.css.styl]: Done.

**Pull request merged**